Source/RapyutaSimulationPlugins/Private/Drives/RRPhysicsJointComponent.cpp

```diff
@@ -256,19 +252,54 @@ void URRPhysicsJointComponent::UpdateState(const float DeltaTime)
 #endif
 }
 
+void URRPhysicsJointComponent::UpdateVelocityTargetFromPose(const FVector InPositionDiff, const FVector InOrientationDiff)
+{
+    uint8 i; 
+    for (i = 0; i < 3; i++)
+    {
+        LinearVelocityTarget[i] =FMath::Clamp(Kvp * InPositionDiff[i], -LinearVelMax[i], LinearVelMax[i]);
```

Comment:
```suggestion
        LinearVelocityTarget[i] = FMath::Clamp(Kvp * InPositionDiff[i], -LinearVelMax[i], LinearVelMax[i]);
```
seems not checked by pre-commit's clang-format?